ui5/eve7/lib/EveElementsRCore.js

```diff
@@ -1401,11 +1402,12 @@ sap.ui.define(['rootui5/eve7/lib/EveManager'], function (EveManager)
          let ib_len = rnr_data.idxBuff.length;
 
          let fcol = RcCol(psp.fMainColor);
+         let mop = Math.min( 1, 1.3 *(1 - psp.fMainTransparency/100));
```

Comment:
And also here? 

Is it intended?

Reply:
Thanks Sergey!
I vaguely remember 1.3 factor was because of front and back rendering, but I forgot why this is for both RenderCore and Three.js renderer. I will be away for the whole week. I will check and test when I come back.